src/schemathesis/cli/__init__.py

```diff
@@ -235,16 +237,20 @@ class OutputStyle(Enum):
 
     default = output.default.DefaultOutputStyleHandler
     short = output.short.ShortOutputStyleHandler
+    junit_xml = output.junitxml.JunitXMLHandler
 
 
 def execute(
     prepared_runner: Generator[events.ExecutionEvent, None, None],
     workers_num: int,
     show_errors_tracebacks: bool,
     store_network_log: Optional[click.utils.LazyFile],
+    junit_xml: Optional[str],
```

Comment:
I'd also recommend adding this value to `ExecutionContext`, so te output handler can show that JUnit report was saved to some file. The handler itself should be in the handlers list before the output handler